drivers/hmis/spec/models/hmis/ce/sql_expression_translator_spec.rb

```diff
@@ -72,5 +72,45 @@
       result = described_class.call('current_age = ((10 + 5) * (2 + 3))', field_map)
       expect(result.to_sql).to include('((10 + 5) * (2 + 3))')
     end
+
+    context 'with unknown fields' do
```

Comment:
It looks like "unknown_field" is something that's not resolvable using the field maps, is that right? I would think that we should raise an error in that case? it feels like a potential recipe for silent failures, for example if we have a typo on a cded key name or something, it would always pass? Let me know if I'm misunderstanding this!